plugins/exporter.koplugin/note.tpl

```diff
@@ -49,33 +49,39 @@
   end
 }#
 <!DOCTYPE html>
-<meta charset="utf-8"/>
-<div style="width:90%; max-width:600px; margin:0px auto; padding:5px; font-size:12pt; font-family:Georgia">
-  <h2 style="font-size:18pt; text-align:right;">#{= htmlescape(booknotes.title) }#</h2>
-  <h5 style="font-size:12pt; text-align:right; color:gray;">#{= htmlescape(booknotes.author) }#</h5>
-  #{ for  _, chapter in ipairs(booknotes) do }#
-    #{ if chapter.title then }#
-      <div style="font-size:14pt; font-weight:bold; text-align:center; margin:0.5em;"><span>#{= htmlescape(chapter.title) }#</span></div>
-    #{ end }#
-    #{ for index, clipping in ipairs(chapter) do }#
-      <div style="padding-top:0.5em; padding-bottom:0.5em;#{ if index > 1 then }# border-top:1px dotted lightgray;#{ end }#">
-        <div style="font-size:10pt; margin-bottom:0.2em; color:darkgray">
-          <div style="display:inline-block; width:0.2em; height:0.9em; margin-right:0.2em; background-color:#{= timecolor(clipping.time)}#;"></div>
-          <span>#{= os.date("%x", clipping.time) }#</span><span style="float:right">#{ if clipping.chapter then }#<b>#{= clipping.chapter }#</b>: #{ end }# #{= clipping.page }#</span>
-        </div>
-        <div style="font-size:12pt">
-          <span>#{= newline_to_br(htmlescape(clipping.text)) }#</span>
-          #{ if clipping.image then }#
-            <en-media type="image/png" hash="#{= clipping.image.hash }#"/>
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+  <meta charset="utf-8"/>
+  <title>#{= htmlescape(booknotes.title) }#</title>
+</head>
+<body>
+  <div style="width:90%; max-width:600px; margin:0px auto; padding:5px; font-size:12pt; font-family:Georgia">
+    <h2 style="font-size:18pt; text-align:right;">#{= htmlescape(booknotes.title) }#</h2>
+    <h5 style="font-size:12pt; text-align:right; color:gray;">#{= htmlescape(booknotes.author) }#</h5>
+    #{ for  _, chapter in ipairs(booknotes) do }#
+      #{ if chapter.title then }#
+        <div style="font-size:14pt; font-weight:bold; text-align:center; margin:0.5em;"><span>#{= htmlescape(chapter.title) }#</span></div>
+      #{ end }#
+      #{ for index, clipping in ipairs(chapter) do }#
+        <div style="padding-top:0.5em; padding-bottom:0.5em;#{ if index > 1 then }# border-top:1px dotted lightgray;#{ end }#">
```

Comment:
Somewhat related to the above comment, crengine supports nth-child thanks to @poire-z's hard work. (Not necessary to use that since I think Netsurf might not, but in any case it'd be more comprehensible if it were just two different classes.)